### ISSUE TITLE
Search filter for stac collection and item level 

### ISSUE THREAD

**Issue by xinli-cai** [Canadian-Geospatial-Platform/stac-to-geocore#5]
This is a feedback from the Tuesday demo related to the geo.ca search. When a user searches for a STAC record on geo.ca,  the user can turn a filter so that it only shows the collection level results or item level results. @bo-lu any suggestions?

**Comment by bo-lu** [comment]
Yes, I believe we created a flag in geocore to indicate the "source system". For example, we currently only have FGP datasets but can have other systems like STAC.

**Comment by xinli-cai** [comment]
Great! I found the 'sourceSystemName' in the geocore template. For STAC, maybe we can have it as STAC-catalog, STAC-collection, and STAC-item?
How does the search pick up the 'sourceSystemName'? I can add the mapping to stac accordingly.

**Comment by bo-lu** [comment]
I would write something like 'CCMEO Datacube STAC - XX'. Where XX is either catalog, collection or item. On the UI side, we can then use regular expression to capture that it is from a 'STAC'

**Comment by xinli-cai** [comment]
I noticed that the current GeoCore on geo.ca does not have the field 'sourceSystemName', see an [example](https://geocore.metadata.geo.ca/042f4628-94b2-40ac-9bc1-ca3ac2a27d82.geojson). I also checked @indiciumx's documentation about [GeoCore format](https://canadian-geospatial-platform.github.io/geocore/docs/geocore-format/), and the 'sourceSystemName' is not included. Can I get some help on the format of this field? Is it a string formatted as 'en; fr'?

**Comment by xinli-cai** [comment]
Or, we can use the 'environmentDescription' field to distinguish FGP HNAP and STAC records? Something like 'environmentDescription': 'en; fr'?
